subsys/shell/Kconfig

```diff
@@ -25,6 +25,17 @@ config SHELL_MINIMAL
 	  defaults which favor reduced flash or memory requirements over extra
 	  features.
 
+config SHELL_THREAD_PRIORITY_OVERRIDE
+    bool
```

Comment:
Without prompt (i.e. `bool "..."`), this symbol is not user configurable (e.g. in `menuconfig`). Please add the prompt as well as `help` describing what this config does.

Reply:
can you check. implemented the change.